mw-webapp/src/component/tooltip/Tooltip.module.scss

```diff
@@ -13,6 +13,10 @@
 
   &:hover + .tooltip {
     display: block;
+    overflow: hidden;
```

Comment:
why you use overflow hidden for tooltip?

Reply:
We need overflow hidden in order to properly truncate text when its too big

Reply:
Then how we can see the whole text?

Reply:
Yea, kinda strange, full text should be visible